texar/modules/decoders/decoder_helpers.py

```diff
@@ -518,6 +537,69 @@ def sample(self, time: int, outputs: torch.Tensor) -> torch.LongTensor:
         return sample_ids
 
 
+class TopPSampleEmbeddingHelper(SingleEmbeddingHelper):
```

Comment:
Pls add unit test, and add the doc to "docs/code/modules.rst"

Reply:
Resolved in d07e236